manifests/init.pp

```diff
@@ -55,6 +55,7 @@
   $conf_template          = $::apache::params::conf_template,
   $servername             = $::apache::params::servername,
   $pidfile                = $::apache::params::pidfile,
+  $rewrite_lock           = false,
```

Comment:
let's make this `undef` by default, since we're dealing with a `String` (path) here.